src/main/scala/uk/gov/hmrc/ct/accounts/approval/boxes/AC199A.scala

```diff
@@ -16,13 +16,17 @@
 
 package uk.gov.hmrc.ct.accounts.approval.boxes
 
-import uk.gov.hmrc.ct.accounts.retriever.AccountsBoxRetriever
+import uk.gov.hmrc.ct.accounts.frs10x.retriever.Frs10xAccountsBoxRetriever
 import uk.gov.hmrc.ct.box.ValidatableBox._
 import uk.gov.hmrc.ct.box._
 
-case class AC199A(value: String) extends CtBoxIdentifier(name = "Approve accounts approver") with CtString with Input with ValidatableBox[AccountsBoxRetriever] {
+case class AC199A(value: String)
+  extends CtBoxIdentifier(name = "Approve accounts approver")
+    with CtString
+    with Input
+    with ValidatableBox[Frs10xAccountsBoxRetriever] {
 
-  override def validate(boxRetriever: AccountsBoxRetriever): Set[CtValidation] = {
+  override def validate(boxRetriever: Frs10xAccountsBoxRetriever): Set[CtValidation] = {
```

Comment:
Just curious more then anything, are we able to be more specific now with the retrievers because we are only doing Frs10x accounts?

Reply:
In theory no. I have done the work for the old accounts as well. In this case the box is on FRS102 and FRS105 so can use FRS10x to validate